static/js/src/static-forms.js

```diff
@@ -123,9 +123,24 @@ function setUpStaticForms(form, formId) {
         message += "\r\n\r\n";
       });
 
+      const radioFieldsets = document.querySelectorAll(
+        ".js-remove-radio-names",
+      );
+      if (radioFieldsets) {
```

Comment:
```suggestion
      if (radioFieldsets) {
```
`querySelectorAll` returns a [NodeList](https://developer.mozilla.org/en-US/docs/Web/API/NodeList), which even when empty returns 'truthy'. So the code inside will always be accessed

Reply:
I've updated it to check for the length of NodeList, thanks for the catch!